tests/clone_test.go

```diff
@@ -696,8 +699,23 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 						libinstancetype.WaitForVMInstanceTypeRevisionNames(sourceVM.Name, virtClient)
 
 						if toRunSourceVM {
-							By("Starting the VM and expecting it to run")
-							sourceVM = RunVMAndExpectLaunchWithRunStrategy(virtClient, sourceVM, virtv1.RunStrategyAlways)
+							sourceVM.Spec.RunStrategy = ptr.To(virtv1.RunStrategyAlways)
```

Comment:
why not extract it to a function that can be used everywhere that calls this RunVMAndExpectLaunchWithRunStrategy function?

Reply:
Where would you put this function?

Reply:
Maybe we want an equivalent of https://github.com/kubevirt/kubevirt/blob/53f1c51472840a3e30a151392ff0b0a1737a4f2f/tests/libkvconfig/kvconfig.go#L37-L49 for VMs?

Reply:
If this sounds like overkill we can just change this test to create a VM that already has the correct runstrategy upfront

Reply:
@akalenyu Sounds good. I think this kind of VM-patch function can live in the future `libclient` package.